src/helpers/transformWeatherData.js

```diff
@@ -137,3 +139,29 @@ export const transformHourlyInfo = (singleHourDataRaw, timezoneOffset) => {
         weatherImage: weatherImage,
     };
 };
+
+export const transformAlertsData = (weatherInfo, timezoneOffset) => {
+    if (!weatherInfo.alerts) {
```

Comment:
What if weatherInfo.alerts is not an array or is empty array?

Reply:
I created it like this, since if there are no alerts, the API returns the current object with no alerts property (making it undefined and falsy). For empty arrays I'd check length, since they're truthy.

Do I need to check for each case, even if I expect the API call to return specific results?